docs/integrations/sources/mysql.md

```diff
@@ -226,6 +241,7 @@ WHERE actor_definition_id ='435bb9a5-7887-4809-aa58-28c27df0d7ad' AND (configura
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                                       |
 |:--------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|
+| 1.0.1  | 2022-09-29 | [17170](https://github.com/airbytehq/airbyte/pull/17170)   | Make Debezium timeout configurable                                                                                                            |
```

Comment:
This changelog is for users to read. `Debezium` is likely a jargon that they have never heard of, and this doc never mentions it. I think the following is slightly easier to understand because keywords like `CDC` and `initial waiting time` are mentioned above in the doc. So there are at least some context.

```suggestion
| 1.0.1  | 2022-09-29 | [17170](https://github.com/airbytehq/airbyte/pull/17170)   | Make initial CDC waiting time configurable |
```